### PR TITLE
Fix tests the fail at the beginning of the cycle

### PR DESCRIPTION
**Pull request by elceebee**
## Context

Now that we are in the 2025 cycle, we need to have 2027 dates in our code base now so that our test suite can find enough future dates to pass when we set the offset to 'after_apply_opens'.

## Changes proposed in this pull request

- Add 2027 tentative dates to CYCLE_DATES
- Define HESA disability codes for 2026-27
- Fix inactive date calculation test by configuring holidays
- Remove some unnecessary time travel

## Guidance to review

This should fix the test suite so we can start to merge things again.

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)
- [ ] Inform data insights team due to database changes
- [ ] Make sure all information from the Trello card is in here
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Add PR link to Trello card

### CODE REVIEW

**Review by tomas-stefano — APPROVED** [review]
🚀

File: config/initializers/cycle_timetables.rb [hunk 1]
```diff
@@ -53,7 +38,24 @@
...
-      easter: Date.new(2026, 3, 18)..Date.new(2026, 4, 1), # TBD
+      easter: Date.new(2026, 3, 23)..Date.new(2026, 4, 6), # TBD
+    },
+  },
+
+}.freeze
+
+FUTURE_TENTATIVE_DATES = {
+  2027 => {
+    find_opens: Time.zone.local(2026, 10, 1, 9), # TBD
```
Comment (avinhurry) [inline config/initializers/cycle_timetables.rb:49]: Do we want to add the notes on here for future us? E.g `# First Tuesday of October`

**Review by avinhurry — APPROVED** [review]
Looks good. One suggestion, not a blocker

File: config/initializers/cycle_timetables.rb (hunk 1 above)
Comment (elceebee) [reply config/initializers/cycle_timetables.rb:49]: Nah, It's elsewhere if we need it. The 2027 dates are definitely just placeholders -- the first tuesday of October in 2027 is actually 6 October.

**Pull request merged**